ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html

```diff
@@ -38,6 +41,7 @@
       data-cmp-page-lang="${container.containingPageLang}"
       data-cmp-path="${resource.path}"
       class="cmp-adaptiveform-container cmp-container ${wcmmode.edit ? 'cmp-adaptiveform-container--edit' : ''}"
+      data-cmp-custom-functions-module-url="${configurationProvider.customFunctionModuleUrl}"
```

Comment:
Please update the component read me about this new data attribute and its purpose

Reply:
done